ephios/user_management/models.py

```diff
@@ -31,6 +31,7 @@ def create_user(
         date_of_birth,
         password=None,
     ):
+        # pylint: disable=unexpected-keyword-arg,no-value-for-parameter
         user = self.model(
             email=email,
             first_name=first_name,
```

Comment:
Here, we are assuming `self.model` is `UserProfile`, but it doesn't has to be, which is why pylint complained I guess, just not on my local machine :/